src/api/action-cable.js

```diff
@@ -6,8 +6,17 @@ import store from '../state';
 
 export const clientId = uuidv4();
 export const task = window.location.pathname.slice(1);
-const password = prompt('Enter judge password');
-const url = `${process.env.REACT_APP_API_ENDPOINT}?password=${password}&task_id=${task}&client_id=${clientId}`;
+var url = `${process.env.REACT_APP_API_ENDPOINT}?task_id=${task}&client_id=${clientId}`;
```

Comment:
Do not use `var`. Use `let` or `const` instead.

```suggestion
let url = `${process.env.REACT_APP_API_ENDPOINT}?task_id=${task}&client_id=${clientId}`;
```